Nix/Parser/Library.hs

```diff
@@ -3,6 +3,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 module Nix.Parser.Library ( module Nix.Parser.Library, module X) where
 
+import Prelude
```

Comment:
Hmm, why is this neccessary? If it's to avoid AMP warnings, I thought that only works well if the import of Prelude is the last import?